extensions/schema-compare/src/test/testSchemaCompareMainWindow.ts

```diff
@@ -66,9 +64,20 @@ export class SchemaCompareMainWindowTest extends SchemaCompareMainWindow {
 			this.selectTargetButton.enabled === buttonState.selectTargetButtonState &&
 			this.generateScriptButton.enabled === buttonState.generateScriptButtonState &&
 			this.applyButton.enabled === buttonState.applyButtonState) {
-			result = true;
+			return true;
 		}
 
-		return result;
+		console.log('CompareButton: (Actual) ', this.compareButton.enabled, '(Expected) ', buttonState.compareButtonState);
```

Comment:
it's a little strange that these are console.logs instead of should

Reply:
Fixed it to use "should" and also printing all the values when it fails.